data/styles/vehicle/include/Subways.mapcss

```diff
@@ -842,7 +842,8 @@ node|z17-[railway=station][transport=subway][city=tianjin] {icon-image: subway-t
 node|z16-[railway=subway_entrance][city=tianjin] {icon-image: subway-tianjin-s.svg;}
 
 /* Tokyo Subway Station */
-node|z13[railway=station][transport=subway][city=tokyo] {icon-image: subway-tokyo-s.svg;icon-min-distance: 1;}
+node|z13[railway=station][transport=subway][city=tokyo]::int_name,
+{icon-image: none;text: none;}
```

Comment:
As you can see in `drules_proto_vehicle_clear.txt` diff, it removed only the secondary label (which is declared via ::int_name) and reset the icon to a generic one.
The right solution would be 
```
node|z13[railway=station][transport=subway][city=tokyo] {icon-image: none; text: none;}
node|z13[railway=station][transport=subway][city=tokyo]::int_name {text: none;}
```
First line removes the icon and primary label, second line removes the secondary label.

Reply:
Also note that you can test vehicle style changes via typing `?vlight` in the search to switch the style and `?light` to switch it back to the main style.